web/content/js/nav.js

```diff
@@ -67,15 +58,57 @@ let navItems = [
 		href: "/Application_Centric_Deployment/index-foreman-el.html"
 	}]
 },{
-	title: "Katello documentation", items: [
+	title: "Foreman on Debian", items: [
+	{
+		title: "Release Notes",
+		href: "/Release_notes/index-foreman-deb.html"
+	},{
+		title: "Quickstart Guide",
+		href: "/Quickstart_Guide/index-foreman-deb.html"
+	},{
+		title: "Installation Guide",
+		href: "/Installing_Server_on_Red_Hat/index-foreman-deb.html"
+	},{
+		title: "Installing Smart Proxy",
+		href: "/Installing_Proxy_on_Red_Hat/index-foreman-deb.html"
+	},{
+		title: "Upgrading and Updating",
+		href: "/Upgrading_and_Updating/index-foreman-deb.html"
+	},{
+		title: "Deploying Foreman on AWS",
+		href: "/Deploying_on_AWS/index-foreman-deb.html"
+	},{
+		title: "Configuring Smart Proxies with a Load Balancer",
+		href: "/Configuring_Load_Balancer/index-foreman-deb.html"
+	},{
+		title: "Provisioning Guide",
+		href: "/Provisioning_Guide/index-foreman-deb.html"
+	},{
+		title: "Configuring Foreman to use Ansible",
+		href: "/Configuring_Ansible/index-foreman-deb.html"
+	},{
+		title: "Managing Hosts Guide",
+		href: "/Managing_Hosts/index-foreman-deb.html"
```

Comment:
This page looks empty to me. Is that local to me?

Reply:
Again, perhaps result of my patch that hides guides which were rendering incorrectly in some contexts. I will remove this Debian menu entry and if @maximiliankolb is interested in bringing it back some time, let's do it!

Reply:
Running Foreman on Debian/Ubuntu is not currently on my plate. However, I would support making the guides/structure more generic/ disconnected from "only" running on RHEL7/8.